### DISCUSSION TITLE
Unable to change modal footer alignment

### DISCUSSION THREAD

**Discussion by SoftCircuits** [twbs/bootstrap#41093] [General] [answered]
If I try to center my footer buttons in a modal popup like this:

     <div class="modal-footer text-center">

The buttons remain right-aligned.

I've tried everything else, even `ms-auto`, but nothing seems to change the right alignment.

How can this be accomplished?

**Comment by julien-deramond** [comment, marked as answer]
Have you tried to add the `justify-content-center` class to this footer?

```html
<div class="modal-footer justify-content-center">
  ...
</div>
```

**Comment by jonhubby** [comment]
I agree with Julien!
